third_party/bls-signatures/build.sh

```diff
@@ -7,7 +7,7 @@ SRC_PATH="$SCRIPT_PATH/src"
 BUILD_PATH="$SCRIPT_PATH/build"
 BLS_SM_PATH="third_party/bls-signatures/src"
 BLS_GIT_REPO="https://github.com/dashpay/bls-signatures.git"
-BLS_GIT_BRANCH="develop_0.1"
+BLS_GIT_BRANCH="develop"
```

Comment:
1. it won't break a release because the release uses the revision which fetched/pulled and committed.
2. opinionated: this parameter was added previously (not by me) and I think we can remove this one because it is impossible not to have submodule in our app